### ISSUE TITLE
remove nmpfit in v 3.4.14

### ISSUE THREAD

**Issue by nden** [spacetelescope/stsci.tools#87]
It was deprecated in 3.4.13. Confirm that there is a release of stistools which does not use nmpfit.

**Comment by nden** [comment]
Yeah, I replaced the code in stistools. Someone else was supposed to enable the mktrace tests. Has that happened yet? @SaOgaz may know.

**Comment by nden** [comment]
@stscirij Actually the PR that replaces this code in stistools has not been merged yet - spacetelescope/stistools#37. it also looks like tests are not running yet.

**Comment by stscirij** [comment]
Ah, ok, I didn't realise that nmpfit was called by gfit.

**Comment by SaOgaz** [comment]
If I'm remembering this correctly, the mktrace tests are in stistools (confirmed this) but I believe Nadia's PR that takes out nmpfit was failing the tests and I wasn't sure why. I believe @nden was out of town at the time.

**Comment by nden** [comment]
stistools hasn't been updated yet. #92 redefined nmpfit will be removed in v 3.5/

**Comment by pllim** [comment]
Maybe we should change the title of this issue to "move gfit and nmpfit to stistools" as per https://github.com/spacetelescope/stsci.tools/pull/138#issuecomment-897029501
